Gruntfile.js

```diff
@@ -146,7 +147,7 @@ module.exports = function(grunt) {
             '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/BasicLatin.js",\n'+
             '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/CombDiacritMarks.js",\n'+
             '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/GeometricShapes.js",\n'+
-            '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/MathOperators.js",\n'+
+            // '"[MathJax]/jax/output/SVG/fonts/TeX/Main/Regular/MathOperators.js",\n'+
```

Comment:
The reason why I commented this file out, along with the others below is because after turning on the `nonull` option, I got warnings for files not present in the MathJax distribution.
When I verify the path, the file is effectively not present in the MathJax 2.7.1 distribution installed from NPM.

Could you please verify?
If the missing files can be pushed then I'll be happy to remove the comments.